install.sh

```diff
@@ -19,6 +19,42 @@ abs_dirname() {
   cd "$cwd"
 }
 
+# performs chmod a+x for all files in $target/$folder 
+# that exist in $src/$folder
+# 
+# Example:
+#     fix_folder_file_permissions $src $target $folder
+#
+fix_folder_file_permissions() {
+  src=$1
+  target=$2
+  folder=$3
+  # scan $src directory for the filenames to fix in $target
+  directory="${src}/${folder}"
+  for f in "${directory}"/*; do
+    # get basename of file (strip the path)
+    filename="${f##*/}"
+    # change the execution bit for the file in $target
+    chmod a+x "${target}/${folder}/$filename"
+  done
```

Comment:
Stylistically, I think it is better to skip `{}` around variable names when not really needed.